### PR TITLE
Better names

### PR DESCRIPTION
**Pull request by bilboquet**
Signed-off-by: Jean-François Morcillo <jfm@massa.net>

### CODE REVIEW

File: proto/abis/massa/abi/v1/abi.proto [hunk 1]
```diff
@@ -566,43 +566,43 @@ message AppendDataRequest {
...
+message DsEntryExistsRequest {
   // key
   bytes key = 1;
   // Address to check data for, if none, use current address
   optional google.protobuf.StringValue address = 2;
 }
 
 // Has data result
-message HasDataResult {
+message DsEntryExistsResult {
```
Comment (Leo-Besancon) [inline proto/abis/massa/abi/v1/abi.proto:605]: When the naming change is important like here, also update the comment.

File: proto/abis/massa/abi/v1/abi.proto [hunk 2]
```diff
@@ -130,53 +130,53 @@ message RespResult {
...
+    GetOpDataResult get_op_data_result = 36;
+    GetOpKeysResult get_op_keys_result = 37;
+    GetOriginOperationIdResult get_origin_operation_id_result = 38;
+    GetOwnedAddressesResult get_owned_addresses_result = 39;
+    GetPubKeyVersionResult get_pub_key_version_result = 40;
+    GetRemainingGasResult get_remaining_gas_result = 41;
+    GetSignatureVersionResult get_signature_version_result = 42;
+    HashBlake3Result hash_blake3_result = 43;
+    HashSha256Result hash_sha256_result = 44;
     HasOpKeyResult has_op_key_result = 45;
```
Comment (Leo-Besancon) [inline proto/abis/massa/abi/v1/abi.proto:165]: `HasDataResult` became `DsEntryExistsResult`, but we're not changing `HasOpKeyResult`

Comment (bilboquet) [reply proto/abis/massa/abi/v1/abi.proto:165]: I dunno…
I asked myself the same question but did not come with a good name, so I did not change it (also it implies change here and in 5 other repos, so…)
Propose a name, i'll do the change

Comment (Leo-Besancon) [reply proto/abis/massa/abi/v1/abi.proto:165]: ```GetDsKeysResult``` VS ```GetOpKeysResult ```    
So 
```DsEntryExistsResult``` VS ```OpEntryExistsResult ```    
?

Comment (bilboquet) [reply proto/abis/massa/abi/v1/abi.proto:165]: done

File: proto/abis/massa/abi/v1/abi.proto (hunk 1 above)
Comment (bilboquet) [reply proto/abis/massa/abi/v1/abi.proto:605]: such a useless comment should be dropped :/

**Pull request merged**